src/schema/objects/modalities.yaml

```diff
@@ -8,6 +8,10 @@ eeg:
   display_name: Electroencephalography
   description: |
     Data acquired with EEG.
+eyetrack:
+  display_name: Eye-tracking
+  description: |
+    Data acquired with an eye-tracker.
```

Comment:
@Remi-Gau I don't think there should be an eyetrack modality, since eyetracking data should always be placed in the corresponding data's modality folder. I would treat it like physio, which doesn't have a modality.

Reply:
Ah, I hadn't thought about the schema validation part. Let me know if you need my help with that (though I think others would know more). Also, sorry for not responding to https://github.com/bids-standard/bids-specification/pull/1128#issuecomment-1312463234 more quickly.

Reply:
I think that we may need to have a mainteners convo about how to handle eyetracking data in the schema and make sure that we tune the validation test to be happy with our decision, rather than the way I was trying to do it (the other way around: modifying the schema to pacify the tests).

May be most productive when all the pieces of the BEP are there but this feels fairly close now.

Reply:
Ok, I took out aspect about modality from the latest version... as it should be, now it has to be checked if it pass the schema validation (not sure what it is)...